polymer/src/sample-category-data.html

```diff
@@ -26,6 +26,11 @@
   <script>
   (function() {
 
+    function sanitizeString(string) {
```

Comment:
What does this method do?

Reply:
It sanitizes strings like this: ""<p>Get started with AMP and learn how to build your first AMP page.</p>\n""